packages/legacy/core/App/components/misc/CredentialCard.tsx

```diff
@@ -19,7 +18,7 @@ interface CredentialCardProps {
   onPress?: GenericFn
   style?: ViewStyle
   proof?: boolean
-  displayItems?: (Attribute | Predicate)[]
+  displayItems?: (legacy.Attribute | legacy.Predicate)[]
```

Comment:
I know you have already made the changes but would it be easier to reference the types at the top of the file. That way you're not having to append everything with `legacy.`

For example:

```
const Attribute = legacy.Attribute;
const Predicate = legacy.Predicate;
...
```

In the future we can still refer to `Attribute`, `Predicate`, etc. but they can be swapped out for modern components without having to replace every instance of `legacy.`

What do you think?

Reply:
That's a nice idea. Future refactoring will be easier indeed.

Reply:
Actually, I don't think it will work so smoothly. Some types it works fine (like `new legacy.DefaultOCABundleResolver` becoming `new DefaultOCABundleResolver`), but enums like `BrandingOverlayType` dont work that fine in parameter type definition for example 
```
  const getCredOverlayType = (type: BrandingOverlayType) => {
```
And I don't think leaving mixed use of this approach may make the code confusing

Reply:
I was actually thinking we may want to export that Branding Overlay Type enum directly. It can be used in the new package code as well. I know it would mean another oca release but its much cleaner.

Reply:
> I was actually thinking we may want to export that Branding Overlay Type Enum directly. It can be used in the new package code as well. I know it would mean another `aries-oca` release but its much cleaner.

Thats a nice idea. It will need another package release, but it's worth the effort. Will open the PR for that.

Reply:
Created the PR #901 and moved the other enums that also might be useful later. As soon this one be merged (and it runs the auto publish), I'll update this PR to use the new references.